### PR TITLE
rune/libenclave/skeleton: Add the usage document of kvmtool

### PR DESCRIPTION
**Pull request by uudiin**
Fixes: #419

Signed-off-by: Tianjia Zhang <tianjia.zhang@linux.alibaba.com>

### CODE REVIEW

File: rune/libenclave/internal/runtime/pal/skeleton/README.md [hunk 1]
```diff
@@ -309,4 +309,55 @@ sudo rune run skeleton-enclave-container
...
+		},
+	}
+}
+```
+
+The configuration of `kvm-rootfs` takes bundle as an absolute address of rootfs, the bundle is specified in the parameters of rune.
+
+```shell
+cp inclavare-containers/pal/skeleton/encl.{bin,elf,ss} bundle/run/rune
+rune --debug run -b bundle test
```
Comment (haosanzi) [inline rune/libenclave/internal/runtime/pal/skeleton/README.md:362]: There is no need to copy `encl.elf`

File: rune/libenclave/internal/runtime/pal/skeleton/README.md [hunk 2]
```diff
@@ -309,4 +309,55 @@ sudo rune run skeleton-enclave-container
...
+		"enclave.runtime.args": "debug backend-kvm kvm-kernel=/path/to/bzImage kvm-rootfs=/ kvm-init=/bin/bash",
+		"enclave.runtime.path": "/path/to/liberpal-skeleton-v2.so",
+	},
+	"linux": {
+		"devices": [
+			{
+				"path": "/dev/kvm",
+				"type": "c",
+				"major": 10,
+				"minor": 232,
```
Comment (hustliyilin) [inline rune/libenclave/internal/runtime/pal/skeleton/README.md:332]: the `minor` number of each machine is different

File: rune/libenclave/internal/runtime/pal/skeleton/README.md [hunk 3]
```diff
@@ -309,4 +309,55 @@ sudo rune run skeleton-enclave-container
...
+				"fileMode": 438,
+				"uid": 0,
+				"gid": 0
+			},
+			{
+				"path": "/dev/net/tun",
+				"type": "c",
+				"major": 10,
+				"minor": 200,
```
Comment (hustliyilin) [inline rune/libenclave/internal/runtime/pal/skeleton/README.md:341]: ditto

**Review by hustliyilin — APPROVED** [review]

**Comment by jiazhang0** [comment]
@uudiin The fix line should be closely followed by SOB line.

**Review by haosanzi — APPROVED** [review]

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**